src/components/Depository/Depository.example.json

```
@@ -0,0 +1,3 @@
+{
+  "content": {}
+}
```

```suggestion
{}
```

можно оставить пустой объект тогда